tests/unit/src/test_conditionSet.cpp

```diff
@@ -76,22 +76,22 @@ SCENARIO( "Extending a ConditionSet", "[ConditionSet][Creation]" ) {
 		REQUIRE( set.IsEmpty() );
 
 		THEN( "no expressions are added from empty nodes" ) {
-			set.Add(DataNode{});
+			set.Load(DataNode{});
 			REQUIRE( set.IsEmpty() );
-			AND_THEN( "a log message is printed to assist the user" ) {
-				REQUIRE( warnings.Flush() == validationWarning );
-			}
+			// AND_THEN( "a log message is printed to assist the user" ) {
```

Comment:
I'm not super happy about these disabled tests.

Reply:
I agree, we should have tests for "empty" and "invalid" conditions, both for ConditionSets and for ConditionAssignments. I was able to restore the test for "invalid" conditions after dealing with some string beautification functionality. Will check later what I can do for the other tests.

Reply:
I restored both checks, but some condition-users actually try to load empty conditions (which now gives an error-message). I'll update those callers not to load the condition when the condition is empty.

Reply:
Also updated the substitutions in conversations and a branch-statement in the data-files that was missing conditions. I believe this is solved now.